client/src/helpers/getRequestAwaitingAdditionalDocuments.js

```diff
@@ -0,0 +1,24 @@
+import getRequestRequirements from 'helpers/getRequestRequirements'
+
+export default (request) => {
+  const {
+    needsIrb,
+    needsMta,
+    shippingRequirement: { needsPayment }
+  } = getRequestRequirements(request.material)
+
+  const additionalDocuments = []
+  if (needsIrb) additionalDocuments.push('IRB')
+  if (needsMta) additionalDocuments.push('signed MTA')
+  if (needsPayment) additionalDocuments.push('shiping payment method')
```

Comment:
Is this only used for new requests? You're pulling these fields off the material, so the current state of the request doesn't impact them at all.

Reply:
![Screen Shot 2020-10-05 at 11 02 06 AM](https://user-images.githubusercontent.com/1075609/95097015-fbbd2600-06fa-11eb-87c6-0372ac9e8b1c.png)

Reply:
That makes sense, although the word `awaiting` in the name seems a bit confusing given that the request hasn't even been made yet.

Reply:
yeah i agree i changed the name of the helper to `getRequesterWillBeAskedToProvide`